zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala

```diff
@@ -67,7 +67,7 @@ trait ZipkinWebFactory { self: App =>
   * we get the host after the [[queryDest]] flag has been parsed.
   */
   lazy val queryClient = new HttpClient(
-    httpService = Http.client.configured(param.Label("zipkin-query"))
+    httpService = Http.client.configured(param.Label("zipkin-web"))
```

Comment:
client annotations should be labeled with the name of the instrumented client. In this case, I mistakenly used the name of the destination. If you looked at the span data before this change, you'd see "cs" labeled with the endpoint zipkin-query. This is incorrect as core annotations should be labeled with the local service name.